setup.py

```diff
@@ -15,13 +15,15 @@ def get_compiler() -> str:
 
 
 def is_64bit_x86() -> bool:
-    arch = platform.machine()
-    return arch in ["x86_64", "x64", "AMD64"]
+    return "SZ_X86_64" in os.environ
+    #arch = platform.machine()
+    #return is_64bit() and arch in ["x86_64", "x64", "AMD64"]
 
 
 def is_64bit_arm() -> bool:
-    arch = platform.machine()
-    return arch in ["arm64", "aarch64", "ARM64"]
+    return "SZ_ARM64" in os.environ
+    #arch = platform.machine()
+    #return is_64bit() and arch in ["arm64", "aarch64", "ARM64"]
```

Comment:
This is may be a good idea when cross-compiling, but it will worsen local builds with `pip install -e .`, and introduces a dependency between `setup.py` and `pyproject.toml`. There should be a way to improve this 🤔

Reply:
I agree, there might be some environment variables that we can check, for either cross compiling or using cibuildwheels.

Reply:
@ashvardanian would you prefer to use an env defined by cibuildwheels (`CIBUILDWHEEL`), a custom env in the `pyproject.toml` Or in the `prerelease.yml`. file?
Also, if its the custom one, what should it be called?

Reply:
I think `cibuildwheel` must define some environment variables, as you've mentioned. It might be the best approach.